### DISCUSSION TITLE
Disable type completions

### DISCUSSION THREAD

**Discussion by kofm** [microsoft/pyright#3383] [Q&A] [answered]
Is it possible to disable ONLY type completions?
ATM I get e.g.:
```python
def get_context_data(self, **kwargs: Any) -> Dict[str, Any]:
    return super().get_context_data(**kwargs)
```
But I would like to get

```python
def get_context_data(self, **kwargs):
    return super().get_context_data(**kwargs)
```

**Comment by erictraut** [comment, marked as answer]
No, this isn't possible currently. Out of curiosity, why would you not want type annotations?

**Reply by kofm** [reply]
Thank you very much for your prompt response and sorry for my late answer.
The fact is that sometimes when I'm prototyping or learning something in Django it is annoying to have to go through all (sometimes long) the list of type hints and import them in the file I'm working.
